doc/changelog/1.18.4-changelog.rst

```diff
@@ -0,0 +1,23 @@
+
+Contributors
+============
+
+A total of 4 people contributed to this release.  People with a "+" by their
+names contributed a patch for the first time.
+
+* Charles Harris
+* Matti Picus
+* Sebastian Berg
+* Warren Weckesser
+
+Pull requests merged
+====================
+
+A total of 6 pull requests were merged for this release.
+
+* `#16055 <https://github.com/numpy/numpy/pull/16055>`__: BLD: add i686 for 1.18 builds
+* `#16090 <https://github.com/numpy/numpy/pull/16090>`__: BUG: random: Generator.integers(2**32) always returned 0.
```

Comment:
Maybe put some double backticks around `2**32`? That may mess up the html rendering.

Reply:
Done. Thanks Ralf.